chat/message/src/MessageLinks/MessageLinks.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { RichLinksArea } from '@lg-chat/rich-links';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { Subtitle } from '@leafygreen-ui/typography';
+
+import {
+  baseStyles,
+  dividingLineStyles,
+  linksHeadingStyles,
+} from './MessageLinks.styles';
+import { type MessageLinksProps } from './MessageLinks.types';
+
+export function MessageLinks({
+  darkMode: darkModeProp,
```

Comment:
If this component is nested inside another component that is wrapped in `<LeafyGreenProvider darkMode={darkMode}>` then you don't need `darkMode` as a prop. The `useDarkMode` hook on L21 should read the darkMode value from `LeafyGreenProvider`

Reply:
However, if this component is intended to also be used outside of `<Message>`, as a standalone component, then that's understandable

Reply:
I wouldn't say it's "intended" but I figured the flexibility would be nice to have should the need ever arise